types/wordpress__block-editor/store/actions.d.ts

```diff
@@ -99,7 +100,7 @@ export function moveBlocksUp(clientIds: string | string[], rootClientId: string)
  * @param start - First block of the multi selection.
  * @param end - Last block of the multiselection.
  */
-export function multiSelect(start: string, end: string): void;
+export function multiSelect(start: string, end: string, __experimentalInitialPosition?: number): void;
```

Comment:
cc: @adamziel - I'm nervous about putting `__experimental` properties in the types, though I understand it's a real pickle since without them there's a type error.

to that end I'm inclined to agree to include them (do the types already contain other experimental properties?) but in doing so, we have to be careful because we'll need to _remove_ them later, as when they standardize or are removed the `__experimental` part will disappear regardless.

Reply:
Yeah, I am on the fence as well... But some of the experimental properties are in the wordpress packages for ages (multiple major versions) without being stabilized, so I don't know how experimental they really are...

Reply:
that's a fairly hot topic 😉

but they are definitely experimental in that it's not considered a breaking change to remove them. for better or worse, people do remove experimental code without trying to prevent breakage of the integrations relying on them.